src/basic/hexdecoct.c

```diff
@@ -89,6 +89,9 @@ int unhexmem(const char *p, size_t l, void **mem, size_t *len) {
         if (l == (size_t) -1)
                 l = strlen(p);
 
+        p = delete_chars(strndupa(p, l), WHITESPACE);
+        l = strlen(p);
```

Comment:
I figure this works too, but this means we'll go through the string twice, once for removing whitespace, and wants for translating to binary... I figure eventually it might make sense to optimize that so that we skip whitespace automatically within the translation loop